android/src/main/kotlin/im/nfc/flutter_nfc_kit/FlutterNfcKitPlugin.kt

```diff
@@ -19,6 +20,10 @@ import org.json.JSONObject
 import java.io.IOException
 import java.util.*
 import kotlin.concurrent.schedule
+import android.os.Looper
```

Comment:
The import statements should be organized (move this to the top section of Android APIs)